src/helpers.js

```diff
@@ -1,18 +1,22 @@
 /* eslint no-console:0 */
 const {allowUnsafeNewFunction} = require('loophole')
 const formatPrettierESLint = require('prettier-eslint')
+const minimatch = require('minimatch')
 
 module.exports.format = format
 module.exports.formatOnSaveIfEnabled = formatOnSaveIfEnabled
 
-function format(event, options = {ignoreSelection: false}) {
+function format(
+  event,
+  options = {ignoreSelection: false},
+  editor = atom.workspace.getActiveTextEditor(),
```

Comment:
Allows passing the `editor` object from `formatOnSave`. I'm not sure whether this is necessary or not. I'm worried that somehow the editor that `formatOnSave` was using to get the path to check globs might change by the time `format` gets called, so this way would prevent that, if that's even an issue.